Main.java

```diff
@@ -0,0 +1,56 @@
+
+public class Main {
+
+	public static void main(String[] args) {
+
+		// Test with no islands
+		Map map = new Map(new boolean[][] { { false, false, false, false }, { false, false, false, false },
```

Comment:
Spreading the map definitions over multiple lines would make them more readable.